ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable.yaml

```diff
@@ -187,6 +187,17 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: osd-ccs-gcp-xpn
+- as: gcp-ocm-osd-ccs-xpn-private-marketplace-f14
```

Comment:
About the period time for the jobs, let us follow up the rules defined in https://github.com/openshift/release/pull/50376(I will record it in a document in future):
4.12. 28 28
4.13. 28 14
4.14. 14 14
4.15. 14 7
4.16. 7 7